fe/fe-core/src/main/java/com/starrocks/catalog/DictionaryMgr.java

```diff
@@ -146,6 +147,20 @@ public void scheduleTasks() {
         }
     }
 
+    public static void fillBackendsOrComputeNodes(List<TNetworkAddress> nodes) {
+        List<Backend> backends = GlobalStateMgr.getCurrentState().getNodeMgr().getClusterInfo().getBackends();
+        for (Backend backend : backends) {
+            nodes.add(backend.getBrpcAddress());
+        }
+
+        if (RunMode.isSharedDataMode()) {
```

Comment:
`compute node` is supported in shared-nothing mode, we should also support this?

Reply:
Fixed